source/nodejs/adaptivecards/src/enums.ts

```diff
@@ -109,6 +109,13 @@ export class ContainerStyle {
     static readonly Emphasis = "emphasis";
 }
 
+export class InputTextStyle {
```

Comment:
Needs to be defined as an enum. ContainerStyle is defined as a class because the style property of Container is a string that can have values that are not defined in ContainerStyle.

Reply:
Got it. On it.